vendor/bat-native-ads/src/bat/ads/internal/base/strings/README.md

```diff
@@ -0,0 +1,3 @@
+# Base String Functions
+
+This set of files contains functions that allow for the streamlined manipulations of strings.
```

Comment:
We only need to add README.md where we give examples of usage etc. Not sure it is needed here